source: http://cgit.openembedded.org/openembedded-core/tree/meta/recipes-extended/quota/quota/fcntl.patch?id=6da9a5269782923fe5807b07217ffe19b9fb633e

Include fcntl.h to pacify compiler errors on musl
like

error: unknown type name 'loff_t'
Cover rpc headers under proper conditional
Dont use __P its undefined

--- quota.h
+++ quota.h
@@ -3,6 +3,7 @@
 
 #include <sys/types.h>
 #include <stdint.h>
+#include <fcntl.h>
 
 typedef u_int32_t qid_t;	/* Type in which we store ids in memory */
 typedef int64_t qsize_t;	/* Type in which we store size limitations */
@@ -181,6 +182,6 @@
 	#endif
 #endif
 
-long quotactl __P((int, const char *, qid_t, caddr_t));
+long quotactl (int, const char *, qid_t, caddr_t);
 
 #endif /* _QUOTA_ */
